libs/ett/core/shell/src/lib/ett-shell.component.ts

```diff
@@ -35,27 +47,129 @@ const selector = 'ett-shell';
     `
       ${selector} {
         display: block;
+
+        .toolbar {
+          width: 100%;
+          display: flex;
+
+          h1 {
+            flex: auto;
+          }
+
+          .menu {
+            align-self: flex-end;
+          }
+        }
       }
     `,
   ],
   template: `
-    <watt-shell>
+    <div *ngIf="profileLoading">Loading</div>
+
+    <watt-shell *ngIf="!profileLoading">
```

Comment:
suggestion: the loading state could be handled using the `*rxLet` structural directive similar to the `EttAuthenticationLinkComponent`